1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -20,6 +20,15 @@
 function pickFromArray(choices) {
   return choices[Math.floor(Math.random() * choices.length)];
 }
+let title = document.querySelector("#title");
+let author = document.querySelector("#author");
+let button = document.querySelector("#btn");
+button.addEventListener("click", () => {
+  let text = pickFromArray(quotes);
+  title.textContent = text.quote;
+  author.textContent = text.author;
+});
+
```

Comment:
well done the new quote functionality works well. Could you think of how you might get a quote to already be there on page load?